data-processing-lib/python/src/data_processing/utils/transform_utils.py

```diff
@@ -144,8 +145,18 @@ def convert_binary_to_arrow(data: bytes, schema: pa.schema = None) -> pa.Table:
             table = pq.read_table(reader, schema=schema)
             return table
         except Exception as e:
-            logger.error(f"Failed to convert byte array to arrow table, exception {e}. Skipping it")
-            return None
+            logger.warning(f"Could not convert bytes to pyarrow: {e}")
+
```

Comment:
Can you, please put a comment here about why polars. Just copy the blur from the where you found this solution

Reply:
Done.